robot/basestation/templates/elements/controls/PdsControls.html

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="command-listener-btn">
       <span>
-        <strong>Command Listener</strong>
+        <strong class"pds-text">Command Listener</strong>
```

Comment:
looks like you're missing the `=` operator here as well as for every other instance in this file

Reply:
oof

Reply:
This hasn't been addressed

Reply:
Missed a couple. Now fixed